tests/endtoend/test_eventhub_batch_functions.py

```diff
@@ -120,7 +121,8 @@ def test_eventhub_multiple_with_metadata(self):
             enqueued_time = parser.isoparse(sys_props['EnqueuedTimeUtc'])
 
             # Check event trigger time and other system properties
-            self.assertTrue(start_time < enqueued_time < end_time)
+            self.assertTrue(
+                start_time.timestamp() < enqueued_time.timestamp() < end_time.timestamp())  # NoQA
```

Comment:
This comparison was happening between timezone aware and timezone naive values, which caused the tests to break. Fixed this by explicitly adding the timezone awareness in `enqueued_time`.